tasks/DisableCpuFreqScaling.rb

```diff
@@ -0,0 +1,16 @@
+class DisableCpuFreqScaling
+  include Genesis::Framework::Task
+
+  init do
+    log "Make sure acpi-cpufreq is enabled" 
+    run_cmd('modprobe acpi-cpufreq')
```

Comment:
I might just be ignorant about this but I messed around with cpupower service and it still would fail until I manually ran this command. I tried to see if installing cpufrequtils would possibly do this as well but it did not :-1: